protocol/synthetix/test/integration/modules/core/IssueUSDModule.burnUSD.test.ts

```diff
@@ -0,0 +1,287 @@
+import assertBn from '@synthetixio/core-utils/utils/assertions/assert-bignumber';
+import assertRevert from '@synthetixio/core-utils/utils/assertions/assert-revert';
+import assertEvent from '@synthetixio/core-utils/utils/assertions/assert-event';
+import { snapshotCheckpoint } from '@synthetixio/core-utils/utils/mocha/snapshot';
+import { BigNumber, constants, ethers } from 'ethers';
+import hre from 'hardhat';
+import { bn, bootstrapWithStakedPool } from '../../bootstrap';
+import { verifyUsesFeatureFlag } from '../../verifications';
+
+const MARKET_FEATURE_FLAG = ethers.utils.formatBytes32String('registerMarket');
+
+describe('IssueUSDModule', function () {
```

Comment:
each shold have a different name to ease logs reading and isolation test runs

Reply:
Yeah good catch, I've missed this one here